physionet-django/user/templates/user/credential_application.html

```diff
@@ -16,7 +16,6 @@ <h1>Application for Credentialed Access</h1>
 
   <p>Please use the form below to apply for {{ SITE_NAME }} credentialing. In order to apply:</p>
   <ul>
-    <li>Complete the CITI Program in "Data or Specimens Only Research", an online course that covers ethics of human research and patient privacy. Instructions are provided <a href="{% url 'citi_course' %}">here</a>.</li>
```

Comment:
This sentence includes a link to guidance for completing the CITI training. Without the link, people will get stuck, so I think we need to keep it. Perhaps it could be displayed on the top of the training page at http://localhost:8000/settings/training/ ?

Reply:
![image](https://user-images.githubusercontent.com/44640544/155983266-c56441a6-c259-4b4e-8727-123cc16e6480.png)